### PR TITLE
Ex. 6 - Counting Items

### PR DESCRIPTION
**Pull request by KinkyHairCoder**
Created an arrow function called countItems to call three other arrow functions called countStrings, countIntegerNumbers, and countDecimalNumbers to count the occurrences of strings, integers, and decimals in the same array.

### CODE REVIEW

File: students/1/ShoppingCart.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+//
```
Comment (mburolla) [inline students/1/ShoppingCart.js:1]: Let's remove the shopping cart file from this PR.

Comment (KinkyHairCoder) [reply students/1/ShoppingCart.js:1]: I think it is removed

Comment (mburolla) [reply students/1/ShoppingCart.js:1]: I still see it.  You should be able to delete the file, then do a new commit, and push the branch.

Comment (KinkyHairCoder) [reply students/1/ShoppingCart.js:1]: I deleted it

**Pull request merged**